apps/fluent-tester/src/TestComponents/Avatar/BasicAvatar.android.tsx

```diff
@@ -2,97 +2,45 @@ import React, { FunctionComponent } from 'react';
 import { Avatar } from '@fluentui-react-native/avatar';
 import { View } from 'react-native';
 import { steveBallmerPhotoUrl } from './../PersonaCoin/styles';
+import { mobileStyles } from '../Common/styles';
 
 export const StandardUsage: FunctionComponent = () => {
   return (
-    <View style={{ paddingBottom: 40 }} >
-      <View style={{
-        flexDirection: 'row',
-        marginHorizontal: 10,
-        paddingTop: 10,
-        paddingBottom: 5,
-        flex: 1,
-        justifyContent: 'space-evenly',
-      }}>
-
+    <View style={{ paddingBottom: 40 }}>
+      <View style={mobileStyles.testVariants}>
         {/* Variation - Standard */}
 
         {/* With badge. */}
-        <Avatar
-          badge={{ status: 'available' }}
-          avatarColor={'brand'}
-          size={40}
-        />
+        <Avatar badge={{ status: 'available' }} avatarColor={'brand'} size={40} />
 
         {/* Variation - No Ring and badge. */}
-        <Avatar
-          size={56}
-          avatarColor={'brand'}
-        />
+        <Avatar size={56} avatarColor={'brand'} />
 
         {/* Variation - With Ring.*/}
-        <Avatar
-          active="active"
-          activeAppearance="ring"
-          size={56}
-          avatarColor={'brand'}
-        />
+        <Avatar active="active" activeAppearance="ring" size={56} avatarColor={'brand'} />
       </View>
 
-      <View style={{
-        flexDirection: 'row',
-        marginHorizontal: 10,
-        paddingTop: 10,
-        paddingBottom: 5,
-        flex: 1,
-        justifyContent: 'space-evenly'
-      }}>
-
+      <View style={mobileStyles.testVariants}>
         {/* Variation - Standard Inverted */}
 
         {/* With badge only.*/}
-        <Avatar
-          size={56}
-          badge={{ status: 'available' }}
-          avatarColor={'brandInverted'}
-        />
+        <Avatar size={56} badge={{ status: 'available' }} avatarColor={'brandInverted'} />
 
         {/* No ring and badge.*/}
-        <Avatar
-          size={56}
-          avatarColor={'brandInverted'}
-        />
+        <Avatar size={56} avatarColor={'brandInverted'} />
 
         {/* With ring.*/}
-        <Avatar
-          active="active"
-          activeAppearance="ring"
-          size={56}
-          avatarColor={'brandInverted'} />
+        <Avatar active="active" activeAppearance="ring" size={56} avatarColor={'brandInverted'} />
       </View>
 
-      <View style={{
-        flexDirection: 'row',
-        marginHorizontal: 10,
-        paddingTop: 10,
-        paddingBottom: 5,
-        flex: 1,
-        justifyContent: 'space-evenly',
-      }}>
+      <View style={mobileStyles.testVariants}>
         {/*Variation - Anonymous*/}
 
         {/* With Badge.*/}
-        <Avatar
-          size={56}
-          badge={{ status: 'available' }}
-        />
+        <Avatar size={56} badge={{ status: 'available' }} />
```

Comment:
There's one comment left from Ruriko about useMemo for objects. It should be an object for the badge e.g.
 `const badgeAvailable = useMemo(
    () => ({
      status: 'available' as PresenceBadgeStatus
    }),
    [],
  );`
P.S. I'll need to update the same for my examples :)